phpstan.neon

```diff
@@ -36,3 +37,8 @@ parameters:
         - '/Symfony\\Component\\ExpressionLanguage\\ParserCache\\ParserCacheInterface/'
         - '/Symfony\\Component\\ExpressionLanguage\\ParserCache\\ParserCacheAdapter/'
         - '/Class Symfony\\Component\\Config\\Definition\\Builder\\TreeBuilder does not have a constructor and must be instantiated without any parameters/'
+
+        # Symfony null typehints
+        - '/Cannot call method arrayNode\(\) on Symfony\\Component\\Config\\Definition\\Builder\\NodeParentInterface\|null/'
```

Comment:
I understand why you are doing this. For the long term I'd prefer to remove these here and instead "fix" the code that is causing problems. If there is an annotation that states "null ist possible" then I think we should handle the case. If in fact the null case is not possible, we can safely do a "if is null throw exception" handling.

Reply:
I agree. I will open an issue for this, so we can look at it after this PR is merged.

Reply:
In this case I gained confidence after seeing that this class was already added to ignore by Sylius team. Also, the code where `phpstan` throws the error is a node configuration with fluent interface where `->end()` could be something or `null`. And checking for `null` here probably will mess the readability or involve deeper changes. But yes, I agree that supress an error is not fixing it. :smile: